tests/zoltan/zoltan_02.mpirun=2.output

```diff
@@ -0,0 +1,8 @@
+
+DEAL::Coloring
+DEAL::Number of colors used: 4
```

Comment:
it's not enough just to add those extra output files, you need to use in tests
```
Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv, 1);
MPILogInitAll all;
```
and remove `initlog();`

Reply:
otherwise you don't really test the output from each core.

Reply:
ps. `MPILogInitAll all;` is a helper class for tests which in constructor creates output file for each MPI core and make `deallog` use it, and on destruction joins them together into a single output file.